CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### Bug fixes
 
+* [#6107](https://github.com/rubocop-hq/rubocop/pull/6107): Fix indentation of multiline postfix conditionals. ([@jaredbeck][])
```

Comment:
Seems this entry is not the right part of the changelog.

Reply:
hmm clearly it should be under "master (unreleased)", but do you also mean it should be under "Changes"?

Reply:
But I see it's under 0.58.2 :-) Unless something's messed up in the GH diff view.

Reply:
Yes, I know. I'm asking if you want it under "master (unreleased) -> Changes" or do you want it under "master (unreleased) -> Fixed"?